### PR TITLE
HMS-5977 add reason in peer leave events webhook doc

### PR DESCRIPTION
**Pull request by aakashbajaj**

### CODE REVIEW

File: docs/server-side/v2/foundation/webhook.mdx [hunk 1]
```diff
@@ -108,6 +108,7 @@ This event will be send when peer leaves the room
 | role       | `string`     | Role of the user                                   | host                                 |
 | left_at    | `timestamp`  | Timestamp when user left                           | 2020-11-11T17:32:17Z                 |
 | duration   | `number`     | Duration the user spent in the room in seconds     | 36000                                |
+| reason     | `string`     | User leave reason                                  | client request                       |
```
Comment (sagargaonkar) [inline docs/server-side/v2/foundation/webhook.mdx:111]: Description - Reason for the peer leaving, see more details below

File: docs/server-side/v2/foundation/webhook.mdx [hunk 2]
```diff
@@ -132,6 +133,20 @@ This event will be send when peer leaves the room
 }
 ```
 
+### Peer Leave Reason
```
Comment (sagargaonkar) [inline docs/server-side/v2/foundation/webhook.mdx:136]: Make it ####
Also, move it before Sample `peer.leave.success` event

File: docs/server-side/v2/foundation/webhook.mdx (hunk 1 above)
Comment (aakashbajaj) [reply docs/server-side/v2/foundation/webhook.mdx]: fixed

File: docs/server-side/v2/foundation/webhook.mdx (hunk 2 above)
Comment (aakashbajaj) [reply docs/server-side/v2/foundation/webhook.mdx]: fixed

**Review by sagargaonkar — APPROVED** [review]

**Pull request merged**